src/routes/JoinChallangeSuccess.jsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useNavigate } from 'react-router-dom';
+import { Flex} from "antd";
+import Heading from "../components/Heading";
+import CustomButton from "../components/CustomButton";
+import Paragraph from "../components/Paragraph";
+
+
+const JoinChallangeSuccess = () => {
+    const navigate = useNavigate();
+
+    const handleReturnToMenu = () => {
```

Comment:
Will need to merge with this handler. Can you explain what skipModal/fromJoinSuccess state mean here?